perun-base/src/main/java/cz/metacentrum/perun/core/api/MemberCandidate.java

```diff
@@ -0,0 +1,69 @@
+package cz.metacentrum.perun.core.api;
+
+
+/**
+ * Candidate with member and rich user of a Virtual Organization.
+ *
+ * @author Vojtech Sassmann &lt;vojtech.sassmann@gmail.com&gt;
+ */
+public class MemberCandidate {
```

Comment:
Please add ``getBeanName()`` method like it's in normal PerunBeans, it is helpful in GUI, since otherwise we don't know object type. Thank you.